src/main/java/org/folio/edge/oaipmh/utils/OaiPmhOkapiClient.java

```diff
@@ -129,6 +128,6 @@ private String getParametersAsString(MultiMap parameters) {
    * @return URL for corresponding 'verb'
    */
   private String getUrlByVerb(MultiMap parameters) {
-    return okapiURL + getEndpoint(parameters) + getParametersAsString(parameters);
+    return okapiURL + getEndpoint(parameters) + "?" + getParametersAsString(parameters);
```

Comment:
not a big deal, but in the future consider using String.format(...)

Reply:
@craigmcnally,
noted. Thanks.